.github/actions/inject-issue-id-action/action.yaml

```diff
@@ -0,0 +1,98 @@
+---
+# SPDX-License-Identifier: Apache-2.0
+# SPDX-FileCopyrightText: 2024 The Linux Foundation
+
+name: "🎟️ Add JIRA ticket to commit message"
+
+inputs:
+  ISSUE_ID_LOOKUP_JSON:
+    description: "JSON mapping users to JIRA tickets"
+    type: string
+    required: true
+  INJECT:
+    # When injection is enabled and no issue found...
+    # String will be added, output variable set true
+    description: "Whether to inject an issue-id if NOT found"
+    type: boolean
+    required: false
+    default: true
+
+outputs:
+  PRESENT:
+    description: "Whether the required ticket string is present"
+    value: ${{ steps.status.outputs.present }}
+
+runs:
+  using: "composite"
+  steps:
+    - name: Checkout
+      uses: actions/checkout@v4
+      with:
+        ref: ${{ github.head_ref }}
+
+    - name: "Get GIT commit message body"
+      id: commit-message
+      # yamllint disable rule:line-length
+      uses: lfit/releng-reusable-workflows/.github/actions/git-commit-message-action@main
+
+    - name: "Check commit message for ticket/string"
+      shell: bash
+      run: |
+        # Check commit message for ticket/string
+        echo "# Checking Commit Message for JIRA Ticket" >> "$GITHUB_STEP_SUMMARY"
+
+        COMMIT_MESSAGE="${{ env.COMMIT_MESSAGE }}"
+        if [ -z "$COMMIT_MESSAGE" ]; then
+          echo "Commit message body was empty/unpopulated ❌"; exit 1
+        fi
+        if [[ "$COMMIT_MESSAGE" =~ "Issue-ID: " ]]; then
```

Comment:
Not all of our projects use `Issue-ID` as their footer. For instance LFIT repositores use `Issue`. As such, the meta-data tag should be configurable but _default_ to `Issue-ID`.

Reply:
I've now made this configurable and tested the change here:
https://github.com/ModeSevenIndustrialSolutions/test-issue-id/pull/51/commits/a11cc0c0abcd1ad1df74f7cfca9b4bb481e1fef1